src/OSDP.Net/DeviceProxy.cs

```diff
@@ -127,7 +127,9 @@ internal OutgoingMessage GetNextCommandData(bool isPolling)
             return new OutgoingMessage(Address, MessageControl, new NoPayloadCommandData(CommandType.Poll));
         }
 
-        return new OutgoingMessage(Address, MessageControl, command);
+        return command != null
```

Comment:
this is interesting. everything I know about OSDP protocol suggests that there must always be a command for every reply. When/how do we have a reply without sending a command for it? Possibly leave a comment explaining the case where this is true?

Reply:
For now this a fail-safe to ensure that an null command is never sent. It indicates there is probably a root issue that needs to be explored further